examples/redis/main.py

```diff
@@ -0,0 +1,75 @@
+from typing import Annotated
+
+from ape import chain
+from ape.api import BlockAPI
+from ape.types import ContractLog
+from ape_tokens import tokens  # type: ignore[import]
+from taskiq import Context, TaskiqDepends, TaskiqState
+
+from silverback import CircuitBreaker, SilverbackApp, SilverbackStartupState
+
+# Do this to initialize your app
+app = SilverbackApp()
+
+# NOTE: Don't do any networking until after initializing app
+USDC = tokens["USDC"]
+YFI = tokens["YFI"]
+
+
+# Can handle some stuff on startup, like loading a heavy model or something
+@app.on_startup()
+def app_startup(startup_state: SilverbackStartupState):
+    return {"message": "Starting...", "block_number": startup_state.last_block_seen}
+
+
+@app.on_client_startup()
+def client_startup(state):
+    return {"message": "Client started."}
+
+
+@app.on_worker_startup()
+def worker_startup(state: TaskiqState):
+    state.block_count = 0
+    # state.db = MyDB()
```

Comment:
this is the only difference between this example and the previous. can we remove this and condense the examples?

Reply:
yeah, I initially separated them because it looked like they might diverge.  Will combine back into one.